pkg/aws/install.go

```diff
@@ -175,7 +177,8 @@ func InstallCluster(name, releaseImage, dhParamsFile string, waitForReady bool)
 	log.Infof("Creating Kube API service")
 	apiNodePort, err := createKubeAPIServerService(client, name)
```

Comment:
When the VPN udp->tcp change makes it in, we can also change the vpn load balancer and get rid of the need for the apiNodePort.